test/service_instance_test/service_instance_test.go

```diff
@@ -635,21 +652,21 @@ var _ = DescribeTestsFor(TestCase{
 										})
 
 										AfterEach(func() {
-											ctx = oldCtx
+											ctx.SMWithBasic = newCtx.SMWithBasic
```

Comment:
because after a new context is created, it deletes the platform record, and the old context (in this case ctx) will have outdated credentials that are not known in SMDB